spec/elastic_apm/span/context/destination_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  class Span
+    class Context
+      RSpec.describe Destination do
+        describe '.from_uri' do
+          let(:uri) { URI('http://example.com') }
```

Comment:
perhaps add a path/query to the URI, to show that they don't end up in the destination context